CMakeLists.txt

```diff
@@ -119,8 +119,26 @@ if(NOT CMAKE_CONFIGURATION_TYPES)
 endif()
 
 ### Compile- and install-related commands.
+# This property will be filled by the CMakeLists of each library with their
+# target name. The target list will be used to create and install CMake
+# configuration files. We define one property in the global scope to be used
+# with the set_property() and get_property() commands.
+# This is also useful to associate documentation with property names that may be
+# retrieved with the get_property() command in subdirectories.
+define_property(GLOBAL
```

Comment:
:-1: -1 for using properties to remember the targets to pass to `install_basic_package_files`...
As already discussed in person, using `EXPORT` is the right way to do it, and `TARGETS` unofficially deprecated, since it requires a lot more work and could lead to differences between the build tree and the install tree

Reply:
@drdanz ok and thanks for the explanation. I'll update the PR asap and ping you back as soon as it is ready 👍